unity/ComputerVision-UnityClient/Assets/Scripts/NetworkManager.cs

```diff
@@ -49,17 +46,24 @@ public class NetworkManager:MonoBehaviour
         public Text avgServerProcessingTimeText;
         public Text avgFullProcessLatencyText;
         public Text avgNetworkOnlyLatencyText;
+        public Text cloudletLocationText;
         public MobiledgeXWebSocketClient client;
-        public static bool showStats; 
+        public static bool showStats;
+        public List<string> regionsDmeList;
         public enum ConnectionMode
         {
             Rest,
             WebSocket
         }
         public ConnectionMode connectionMode;
+        public static int regionIndex = 0;
+
         #region MonoBehaviour Callbacks
         IEnumerator Start()
         {
+            regionsDmeList = new List<string>() { "wifi", "eu-mexdemo", "us-mexdemo", "jp-mexdemo", "kr-mexdemo" };
```

Comment:
We maintain a list of DMEs at http://dme-inventory.mobiledgex.net/dme-list.html. If you use this list to populate your DME values, then we can easily add a new DME for someone without having to rebuilt the app. This is not an immediate requirement, but a "would be nice".

Reply:
Shouldn't this list be pulled from the Unity SDK Editor? (who will start with a default list, and pull it from a server somewhere else)?

Reply:
@lgarner  you have a point, will update in the next SDK release and then pull it in the app from the SDK.
@brucearmstrong  I was searching for something like that, Thanks.